conf/messages.en

```diff
@@ -166,24 +166,24 @@ convertedPackagingCredit.change.hidden = How much credit would you like to claim
-directlyExportedComponents.title = Did you export any of your {0} kg of finished plastic packaging components in this period yourself, or do you intend to within 12 months?
-directlyExportedComponents.heading = Did you export any of your {0} kg of finished plastic packaging components in this period yourself, or do you intend to within 12 months?
+directlyExportedComponents.title = Did you export any of your {0} of finished plastic packaging components in this period yourself, or do you intend to within 12 months?
+directlyExportedComponents.heading = Did you export any of your {0} of finished plastic packaging components in this period yourself, or do you intend to within 12 months?
 directlyExportedComponents.checkYourAnswersLabel = Exported plastic packaging
 directlyExportedComponents.error.required = Select yes if you exported any of your finished plastic packaging components in this period yourself, or if you intend to within the next 12 months
-directlyExportedComponents.change.hidden = Did you export any of your {0} kg of finished plastic packaging components in this period yourself, or do you intend to within 12 months?
+directlyExportedComponents.change.hidden = Did you export any of your {0}kg of finished plastic packaging components in this period yourself, or do you intend to within 12 months?
 directlyExportedComponents.paragraph.1 = Only include plastic packaging that has been exported by you, or on your behalf.
 directlyExportedComponents.paragraph.2 = You will not be charged tax on these but you must still tell us about them. If you do not export these plastics within 12 months, you’ll need to pay tax on them.
 
-exportedPlasticPackagingWeight.title = How much of your {0} kg of finished plastic packaging components did you export?
-exportedPlasticPackagingWeight.heading = How much of your {0} kg of finished plastic packaging components did you export?
+exportedPlasticPackagingWeight.title = How much of your {0} of finished plastic packaging components did you export?
+exportedPlasticPackagingWeight.heading = How much of your {0} of finished plastic packaging components did you export?
 exportedPlasticPackagingWeight.checkYourAnswersLabel = Weight of exported plastic packaging
 exportedPlasticPackagingWeight.error.nonNumeric = Weight must be entered as numbers only
 exportedPlasticPackagingWeight.error.required = Enter the weight, in kilograms
 exportedPlasticPackagingWeight.error.wholeNumber = Weight must not include decimals
 exportedPlasticPackagingWeight.error.spaces = Weight must not include spaces
 exportedPlasticPackagingWeight.error.outOfRange.low = Weight must be 0kg or more
 exportedPlasticPackagingWeight.error.outOfRange.high = Weight must be 99,999,999,999kg or less
-exportedPlasticPackagingWeight.change.hidden = How much of your {0} kg of finished plastic packaging components did you export?
+exportedPlasticPackagingWeight.change.hidden = How much of your {0}kg of finished plastic packaging components did you export?
```

Comment:
shoud this kg be removed two, and the input value just call asKgs

Reply:
change.hidden messages updated with .asKg for the parameter